custom/abdm/milestone_one/views/abha_verification_views.py

```diff
@@ -0,0 +1,75 @@
+from rest_framework.decorators import (
+    api_view,
+    authentication_classes,
+    permission_classes,
+)
+from rest_framework.permissions import IsAuthenticated
+
+from custom.abdm.auth import ABDMUserAuthentication
+from custom.abdm.milestone_one.utils import abha_verification_util as abdm_util
+from custom.abdm.milestone_one.utils.decorators import required_request_params
+from custom.abdm.milestone_one.utils.response_util import (
+    generate_invalid_req_response,
+    parse_response,
+)
+
+
+@api_view(["GET"])
+@permission_classes((IsAuthenticated,))
+@authentication_classes((ABDMUserAuthentication,))
+def get_auth_methods(request):
+    health_id = request.query_params.get("health_id")
+    if not health_id:
+        error_msg = "Missing required parameter in the request param: health_id"
```

Comment:
nit: 
"Missing required parameter in the request params: health_id"
or
"Missing required parameter: health_id" would also suffice

Reply:
https://github.com/dimagi/commcare-hq/pull/32332/commits/fa6af013987a2dba9ca2a933aa8bb7b1c13db5c9